crates/rune/src/options.rs

```diff
@@ -51,25 +51,35 @@ impl Options {
     }
 
     /// Set if debug info is enabled or not. Defaults to `true`.
-    pub fn debug_info(&mut self, enabled: bool) {
+    pub fn debug_info(&mut self, enabled: bool) -> &mut Self {
```

Comment:
Method chaining can be a problematic approach to using an API, see:

https://randompoison.github.io/posts/returning-self/

It hits a few notes that will be relevant here, b so I'd rather you avoid it. And there's no reason to `cascade!` yet either. It's not complex enough.